aio/aio-proxy/aio_proxy/request/helpers.py

```diff
@@ -1,24 +1,37 @@
-from aio_proxy.decorators.value_exception import value_exception_handler
+def match_bool_to_insee_value(bool_value: bool) -> str | None:
+    """Match bool filter value to corresponding INSEE field value .
 
+    Args:
+        bool_value (str): `est_var` value extracted from request
 
-@value_exception_handler(
-    error="Veuillez indiquer une date minimale inférieure à la date maximale."
-)
-def validate_date_range(min_date=None, max_date=None):
-    if min_date and max_date:
-        if max_date < min_date:
-            raise ValueError
+    Returns:
+        None if `est_var`is None.
+        "O" if `est_var` is True.
+        "N" if `est_var` is False.
```

Comment:
est_var vs bool_value

Souhld nt be bool_value : `bool | None` ?

Reply:
It's just bool, since the None use case is treated by Pydantic.